tests/storage/lvm_test.py

```diff
@@ -1217,6 +1217,120 @@ def test_vg_remove_by_uuid(monkeypatch, fake_devices):
     assert len(lc._vgs) == len(fake_vgs) - 1
 
 
+def test_vg_check_block_sizes(monkeypatch, fake_devices):
```

Comment:
The test should be done using real pv and vg and no mocking.

We don't support 4k storage yet - if we will add such support, we can test it using
userstorage.

Reply:
I have removed these for now, we can discuss if it is worth the effort or not to add them in a separate branch.